index.html

```diff
@@ -1,75 +1,425 @@
-          <div class="projects">
-            <div class="project">
-                <div class="logo"><a href="iris/index.html"><img src="images/Iris7_1_trim_100.png" style="height:100;" alt="Iris Logo" /></a></div>
-                <div class="summary">
-                    <p>
-                        <a href="iris/index.html">Iris</a>
-                        is a Python package for analysing and visualising meteorological and oceanographic data sets.
-                        <br />
-                        See the <a href="iris/docs/latest/index.html">latest documentation</a> for Iris.
-                    </p>
-                </div>
+  <section class="header1 index-clouds mbr-parallax-background">
+    <div class="mbr-overlay"
+         style="opacity: 0.4; background-color: rgb(15, 118, 153);"></div>
+    <div class="container">
+      <div class="row justify-content-md-center">
+        <div class="mbr-white col-md-10">
+          <h1 class="mbr-section-title align-center mbr-bold pb-3 mbr-fonts-style display-1">
+            <a class="anchor" name="home-top" id="home-top"></a>
+            SciTools
+          </h1>
+          <h3 class="mbr-section-subtitle align-center mbr-light pb-3 mbr-fonts-style display-2">
+            &#160;Open tools for the analysis and visualisation of Earth science
+            data
+          </h3>
+          <p class="mbr-text align-center pb-3 mbr-fonts-style display-5">
+          SciTools is a collaborative effort to produce powerful Python-based
+          open-source tools for Earth scientists. Initially started at the
+          <a href="https://www.metoffice.gov.uk/"
+             target="_blank"
+             class="text-white">Met Office</a> in 2010, SciTools has grown
+             into a diverse community of partners and collaborators from
+             around the world. SciTools is responsible for the maintenance of
+             a number of key packages such as Iris and Cartopy, and continues
+             to develop new and innovate tools for the Earth scientist&#39;s
+             toolkit.</p>
+          <div class="mbr-section-btn align-center">
+            <a class="btn btn-md btn-primary display-4" href="index.html#iris">
+                <span class="mbri-globe-2 mbr-iconfont mbr-iconfont-btn">
+                </span> Iris
+            </a>
+            <a class="btn btn-md btn-primary display-4" href="index.html#cartopy">
+                <span class="mbri-globe mbr-iconfont mbr-iconfont-btn">
+                </span> Cartopy
+            </a>
+            <a class="btn btn-md btn-primary display-4" href="index.html#additional_tools">
+                <span class="mbri-setting3 mbr-iconfont mbr-iconfont-btn">
+                </span> Additional Tools
+            </a>
+            <a class="btn btn-md btn-primary display-4" href="index.html#cf_units">
+                <span class="mbri-watch mbr-iconfont mbr-iconfont-btn">
+                </span> cf_units
```

Comment:
I misunderstood the purpose of these buttons. I see that they're more of a contents rather links other pages as I expected them to be. This does feel a bit unnatural
Regardless, I would remove cf_units as it isn't an individual section